### ISSUE TITLE
Feature Request: Count repeated key combinations

### ISSUE THREAD

**Issue by joeparis** [Code52/carnac#103]
It would be be great (for me) if repeated key combos were simply counted instead of displayed againd and again, something like what's shown below. 

![prtotype](https://cloud.githubusercontent.com/assets/2662123/11463058/c5327fc0-96d1-11e5-8e88-beafbe446781.png)

**Comment by TechnoTone** [comment]
Yes, I'd like that too. It does this already for the cursor keys without combo's but not for other keys or combos.

**Comment by joeparis** [comment]
I'm glad to hear this would be useful to someone besides me. I may try to take a swing at this and #94 in the next few weeks....

**Comment by JakeGinnivan** [comment]
If you have any issues @joeparis I am more than happy to help. Have created a chat room at https://gitter.im/Code52/carnac, ping me on there

**Comment by joeparis** [comment]
Thanks!
